Source/SplineFollow/Private/Components/SFSplineComponent.cpp

```diff
@@ -59,6 +88,41 @@ void USFSplineComponent::UpdateSpline()
     }
 }
 
+void USFSplineComponent::Serialize( FArchive & archive )
+{
+    if ( archive.IsSaving() )
+    {
+        for ( const auto & action_marker : StaticActionMarkers )
+        {
+            auto marker = FSFSplineMarker( action_marker.Name, action_marker.ItIsEnabled, action_marker.Infos, NewObject< USFSplineMarkerObject_Action >( this ) );
```

Comment:
replace by 

`FSFSplineMarker marker( action_marker.Name, action_marker.ItIsEnabled, action_marker.Infos, NewObject< USFSplineMarkerObject_Action >( this ) );`

to avoid a copy

Reply:
using lambda